client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html

```diff
@@ -1,6 +1,6 @@
 <div class="cui-page__content inputs-logs">
-    <oui-back-button class="back_button_inline_text"></oui-back-button>
-    <span class="oui-header_3" data-translate="inputs_logs_title" data-translate-values="{inputTitle: ctrl.input.data.info.title}"></span>
+    <oui-back-button></oui-back-button>
```

Comment:
The usage of oui-back-button allows to put a data-title inside like that :
`<oui-back-button data-title="{{ 'inputs_logs_title' | translate: {inputTitle: ctrl.input.data.info.title} }}"></oui-back-button>`

Reply:
I tried it and it does not work. I mentioned it in UI-Kit HipChat room but I did not get any response on it. Seems like a bug on oui-back-button component.

Reply:
@AxelPeter have you heard about this ?

Reply:
created a jira task UK-44 to track this issue